src/components/Navbar/Navbar.jsx

```diff
@@ -8,52 +8,47 @@ import { Link, NavLink, withRouter } from "react-router-dom";
 import { NavbarIcon } from "components";
 import { StyledNav, NavIcons, HomeIcon, SearchInput } from "./Navbar.styles";
 
-class Navbar extends React.Component {
-   state = {
-      inputValue: "",
-   };
+const Navbar = (props) => {
+   const [inputValue, setInputValue] = useState("");
 
-   handleChange = (e) => {
-      this.setState({ inputValue: e.target.value });
+   const handleChange = (e) => {
+      setInputValue(e.target.value);
    };
 
-   handleSubmit = (e, inputValue) => {
+   const handleSubmit = (e, inputValue) => {
       e.preventDefault();
 
-      this.props.history.push(`/search/${inputValue}`);
-      this.setState({ inputValue: "" });
+      props.history.push(`/search/${inputValue}`);
+      setInputValue("");
```

Comment:
why do you need to pass inputValue as an argument? you can grab it straight away from line 15